akka-samples/akka-docs-java-lambda/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <publishedAkkaVersion>2.3.7</publishedAkkaVersion>
+    <publishedAkkaVersion>2.3.9</publishedAkkaVersion>
```

Comment:
thanks for clarifying—these things will be cleaned up soon in other ways as well (removing the samples from the main build)